TelegramClient-UI.package/TCUMain.class/instance/addHeader.st

```diff
@@ -13,7 +11,7 @@ addHeader
 	logoutButton := TCUButton new
 	 	text: 'logout';
 		on:#mouseUp send: #logout to: self;
-		position: header position + 5.
+		position: (header position x + header width - 105) @ 5.
```

Comment:
try and remove that 105 by using a `layoutPolicy` as used in `TCUChatsList >> initializeLayout`

Reply:
`layoutPolicy` not working well here because of the centered title